js/stringProcessing/getSentences.js

```diff
@@ -82,20 +98,30 @@ module.exports = function( text ) {
 	var originalText = text;
 
 	// Unify all terminators.
-	text = text.replace( /[.?!]/g, "." );
+	text = text.replace( /[.?!\:;]/g, "." );
```

Comment:
We should put these delimiting characters in a variable so it's easier to see what their purpose is because they are named. You call this variable `SENTENCE_DELIMITERS`.